src/constants/constants.ts

```diff
@@ -18,3 +18,27 @@ export enum CollaboratorRoles {
 export const E2E_ISOMER_ID = "-1"
 export const E2E_TEST_EMAIL = "test@e2e"
 export const E2E_TEST_CONTACT = "12345678"
+
+export const GH_MAX_REPO_COUNT = 100
+export const ISOMERPAGES_REPO_PAGE_COUNT =
+  (process.env.ISOMERPAGES_REPO_PAGE_COUNT &&
+    parseInt(process.env.ISOMERPAGES_REPO_PAGE_COUNT)) ||
```

Comment:
think we missed the radix argument here

Reply:
Whoops TIL about the radix argument. Done in 3658486.